Code-the-Dream-School/ee-prac-team1-front
# PRAC-34 React - Activities container

The activities container component has been created.

Testing Steps:
Run the application.
Verify that activity cards appear.
![image](https://github.com/Code-the-Dream-School/ee-prac-team1-front/assets/109319234/285e7eee-f22e-4482-b815-75f470f6ee09)
